cmd/security-proxy-setup/Dockerfile

```diff
@@ -26,6 +26,7 @@ WORKDIR /edgex
 
 COPY --from=builder /edgex-go/cmd/security-proxy-setup/res/configuration.toml res/configuration.toml
```

Comment:
Where is the configuration.toml for edgex-secrets-config? Is it sharing security-proxy-setup's?

Reply:
Yes, it will share.

Reply:
Please add `why` comment since this is not normal for our service to share configuration.